doc/radosgw/lua-scripting.rst

```diff
@@ -135,6 +135,12 @@ Debug Log
 The ``RGWDebugLog()`` function accepts a string and prints it to the debug log with priority 20.
 Each log message is prefixed ``Lua INFO:``. This function has no return value.
 
+The ``RGWDebugLog2()`` function accepts a format string and a variable list of arguments,
+and prints the formatted string to the debug log with priority 20.
+Instances of "{}" in the format string are replaced by the corresponding arguments.
```

Comment:
please add any restrictions: not escape sequenes, no position arguments and no format specs

Reply:
not sure that the term "positional arguments" is correct here.
where arg-index is used, these are actually non-positional arguments. AFAIK, "positional arguments" are arguments where the order in which they are passed to the function matters.
according to this: https://en.cppreference.com/w/cpp/utility/format/vformat.html
we wan tto avoid:
* arg-id
* format-spec
* escape sequence

IMO, we should use these terms here as well